tgui-next/packages/tgui/interfaces/AtmosControlConsole.js

```diff
@@ -45,10 +45,12 @@ export const AtmosControlConsole = props => {
           <Section
             title="Controls"
             buttons={(
+              data.can_reconnect ? (
               <Button
                 icon="undo"
                 content="Reconnect"
                 onClick={() => act(ref, 'reconnect')} />
+            ) : (null)
             )}>
```

Comment:
Please fix tabbing (with `npm run lint --fix`), or better yet, do it like this, which is cleaner:
```jsx
<Section
  title="Controls"
  buttons={Boolean(data.can_reconnect) && (
    <Button
      icon="undo"
      content="Reconnect"
      onClick={() => act(ref, 'reconnect')} />
  )}>
```

Reply:
Done, used your thing.